samples/managed-app-sample/README.md

```diff
@@ -0,0 +1,155 @@
+
+- User Assigned identity:
+
+```json
+{
+    "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+    "name": "[concat(parameters('clusterResourceName'), '-identity')]",
+    "apiVersion": "2023-01-31",
+    "location": "[resourceGroup().location]"
+},
+```
+
+- Keyvault, access policy and secret:
+
+```json
+{
+    "name": "[uniqueString(resourceGroup().id, resourceGroup().location, 'keyvault')]",
+    "type": "Microsoft.KeyVault/vaults",
+    "apiVersion": "2019-09-01",
+    "location": "[resourceGroup().location]",
+    "tags": {
+        "displayName": "keyVault1"
+    },
+    "properties": {
+        "enabledForDeployment": true,
+        "enabledForTemplateDeployment": true,
+        "enabledForDiskEncryption": true,
+        "tenantId": "[subscription().tenantId]",
+        "accessPolicies": [
+            {
+                "tenantId": "[subscription().tenantId]",
+                "objectId": "[reference(resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', concat(parameters('clusterResourceName'), '-identity')), '2023-01-31').principalId]",
+                "permissions": {
+                    "keys": [
+                        "Get"
+                    ],
+                    "secrets": [
+                        "List",
+                        "Get",
+                        "Set"
+                    ]
+                }
+            }
+        ],
+        "sku": {
+            "name": "standard",
+            "family": "A"
+        }
+    },
+    "resources": [
+        {
+            "type": "secrets",
+            "name": "secretExample1",
+            "apiVersion": "2016-10-01",
+            "dependsOn": [
+                "[resourceId('Microsoft.KeyVault/vaults', uniqueString(resourceGroup().id, resourceGroup().location, 'keyvault'))]"
+            ],
+            "properties": {
+                "value": "secretValue"
+            }
+        }
+    ],
+    "dependsOn": [
+        "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', concat(parameters('clusterResourceName'), '-identity'))]"
+    ]
+}
+```
+
+- Assigning the identity to the cluster's node and depends on:
+
+```json
+"type": "Microsoft.ContainerService/managedClusters",
+"apiVersion": "2022-11-01",
+"name": "[parameters('clusterResourceName')]",
+"location": "[parameters('location')]",
+"dependsOn": [
+    "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', concat(parameters('clusterResourceName'), '-identity'))]"
+],
+"tags": {},
+"sku": {
+    "name": "Basic",
+    "tier": "Free"
+},
+"identity": {
+    "type": "UserAssigned",
+    "userAssignedIdentities": {
+        "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', concat(parameters('clusterResourceName'), '-identity'))]": {}
+    }
+
+},
+```
+
+## UIDefinition
+
+In this sample [createUiDefinition](./createUiDefinition.json), we have remove extra components that is not applicable for the managed app offer.
```

Comment:
nit: we have removed extra components that are not applicable for the managed app offer